### PR TITLE
Adding i18n in Vietnamese

### PR DESCRIPTION
**Pull request by truongnh1992**
Signed-off-by: Nguyen Hai Truong <truongnh@vn.fujitsu.com>

### CODE REVIEW

File: i18n/vi.toml [hunk 1]
```diff
@@ -0,0 +1,188 @@
+# i18n strings for the Vietnamese (main) site.
+
+[caution]
+other = "Chú ý:"
+
+[cleanup_heading]
+other = "Cleaning up"
```
Comment (huynguyennovem) [inline i18n/vi.toml]: Please translate this!

File: i18n/vi.toml [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+other = "Twitter"
+
+[deprecation_warning]
+other = " tài liệu này không còn được duy trì. Phiên bản đang xem hiện tại là một snapshot tĩnh. Để rõ hơn về tài liệu bản cập nhật, xem "
+
+[deprecation_file_warning]
+other = "Không dùng nữa"
+
+[docs_label_browse]
+other = "Browse Docs"
```
Comment (huynguyennovem) [inline i18n/vi.toml]: Please translate this!

File: i18n/vi.toml [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+[docs_label_contributors]
+other = "Người đóng góp"
+
+[docs_label_i_am]
+other = "I AM..."
+
+[docs_label_users]
+other = "Users"
```
Comment (huynguyennovem) [inline i18n/vi.toml]: Please translate this!

File: i18n/vi.toml [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+other = "Bảng chú giải này được kì vọng là một danh sách hoàn thiện, được chuẩn hóa về thuật ngữ Kubernetes. Nó bao gồm các thuật ngữ kỹ thuật dành riêng cho Kubernetes, cũng như các thuật ngữ chung hơn cung cấp ngữ cảnh hữu ích"
+
+[layouts_docs_glossary_deselect_all]
+other = "Bỏ chọn tất cả"
+
+[layouts_docs_glossary_click_details_after]
+other = "danh mục dưới đây để giải thích rõ hơn cho các thuật ngữ cụ thể"
+
+[layouts_docs_glossary_click_details_before]
+other = "Click on the"
```
Comment (huynguyennovem) [inline i18n/vi.toml]: Please translate this!

File: i18n/vi.toml [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+other = "Xem trên GitHub"
+
+[main_kubernetes_features]
+other = "Các tính năng của Kubernetes"
+
+[main_kubeweekly_baseline]
+other = "Muốn nhận những tin tức Kubernetes mới nhất? Đăng kí KubeWeekly."
+
+[main_kubernetes_past_link]
+other = "View past newsletters"
```
Comment (huynguyennovem) [inline i18n/vi.toml]: Please translate this!

File: i18n/vi.toml [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+other = "Mục tiêu"
+
+[prerequisites_heading]
+other = "Trước khi bắt đầu"
+
+[ui_search_placeholder]
+other = "Tìm kiếm"
+
+[version_check_mustbe]
+other = "Your Kubernetes server must be version "
```
Comment (huynguyennovem) [inline i18n/vi.toml]: Please translate this!

File: i18n/vi.toml [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+
+[version_check_mustbeorlater]
+other = "Your Kubernetes server must be at or later than version "
```
Comment (huynguyennovem) [inline i18n/vi.toml]: Please translate this!

File: i18n/vi.toml [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+
+[version_check_tocheck]
+other = "To check the version, enter "
```
Comment (huynguyennovem) [inline i18n/vi.toml]: Please translate this!

**Comment by truongnh1992** [comment]
/assign @jaredbhatti
Could you help me to approve this PR. It has been approved by a owner Vietnamese localization K8s document.

**Comment by truongnh1992** [comment]
ping @jaredbhatti

**Comment by truongnh1992** [comment]
@ngtuna could you help me to review this PR, please.

**Comment by truongnh1992** [comment]
ping @remyleone 
Could you help me to approve this PR. It has been approved by a owner Vietnamese localization K8s document.

File: i18n/vi.toml [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+other = "Khám phá cộng đồng"
+
+[main_contribute]
+other = "Đóng góp"
+
+[main_copyright_notice]
+other = """The Linux Foundation &reg;. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our <a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">Trademark Usage page</a>"""
+
+[main_documentation_license]
+other = """The Kubernetes Authors | Documentation Distributed under <a href="https://git.k8s.io/website/LICENSE" class="light-text">CC BY 4.0</a>"""
```
Comment (remyleone) [inline i18n/vi.toml]: Not translated

File: i18n/vi.toml [hunk 10]
```diff
@@ -0,0 +1,188 @@
...
+other = """We are a <a href="https://cncf.io/">CNCF</a> graduated project</p>"""
+
+[main_community_explore]
+other = "Khám phá cộng đồng"
+
+[main_contribute]
+other = "Đóng góp"
+
+[main_copyright_notice]
+other = """The Linux Foundation &reg;. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our <a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">Trademark Usage page</a>"""
```
Comment (remyleone) [inline i18n/vi.toml]: Not translated

File: i18n/vi.toml [hunk 11]
```diff
@@ -0,0 +1,188 @@
...
+other = "Cảm ơn vì đã phản hồi. Nếu bạn có một câu hỏi cụ thể, có thể trả lời về cách sử dụng Kubernetes, hãy hỏi nó trên"
+
+[layouts_docs_search_fetching]
+other = "Đang lấy kết quả..."
+
+[main_by]
+other = "bởi"
+
+[main_cncf_project]
+other = """We are a <a href="https://cncf.io/">CNCF</a> graduated project</p>"""
```
Comment (remyleone) [inline i18n/vi.toml]: Not translated

File: i18n/vi.toml [hunk 12]
```diff
@@ -0,0 +1,188 @@
...
+other = "Không dùng nữa"
+
+[docs_label_browse]
+other = "Duyệt tài liệu"
+
+[docs_label_contributors]
+other = "Người đóng góp"
+
+[docs_label_i_am]
+other = "I AM..."
```
Comment (remyleone) [inline i18n/vi.toml]: Not translated

**Comment by truongnh1992** [comment]
Thank @remyleone and @huynguyennovem for your review.
I've already resolved all of your comments.

**Comment by truongnh1992** [comment]
/assign @kbarnard10
Could you help me to approve this PR, please. It has been approved by a owner Vietnamese localization K8s document.

**Pull request merged**